congress/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .base import Client, BaseParser, BaseValidator
 from .congress import Congress
+from .votes import Votes
```

Comment:
Please do use the new line at end of file convention.  This can be enabled in most text editors

Reply:
Done. Thanks.